echopype/convert/set_groups_ek80.py

```diff
@@ -909,21 +923,25 @@ def _assemble_ds_common(self, ch, range_sample_size):
                     {
                         "long_name": "Transceiver mode",
                         "flag_values": [0, 1],
-                        "flag_meanings": ["Active", "Inactive"],
+                        "flag_meanings": ["Active", "Unknown"],
                     },
                 ),
-                "pulse_form": (
+                "transmit_type": (
                     ["ping_time"],
-                    np.array(self.parser_obj.ping_data_dict["pulse_form"][ch], dtype=np.byte),
+                    pulse_form_map(np.array(self.parser_obj.ping_data_dict["pulse_form"][ch])),
                     {
-                        "long_name": "Pulse type",
-                        "flag_values": [0, 1, 5],
-                        "flag_meanings": ["CW", "FM", "FMD"],
+                        "long_name": "Type of transmitted pulse",
+                        "flag_values": ["CW", "FM", "FMD"],
+                        "flag_meanings": [
+                            "Continuous Wave",
+                            "Frequency Modulated",
+                            "Frequency Modulated D",
```

Comment:
Should we change this to "Frequency Modulated user-Defined" or do you think this is extrapolating too far from the documentation about ADCP?

Reply:
I don't know. As I mentioned in https://github.com/OSOceanAcoustics/echopype/pull/1101#issuecomment-1657451416, even the description next to LFMD seems scrambled and unclear. Having the acronyms spelled out here, as far as we can, seemed like progress, but I'm hesitant to go further. Still, these are only attribute values that would have no downstream implications, so they can be easily tweaked in the future.

Reply:
Let's leave it as is until we have better information.